recipe/build.sh

```diff
@@ -14,3 +14,55 @@ case $(uname -s) in
         build_linux
         ;;
 esac
+
+
+# If/when https://github.com/conda/conda-build/issues/4121 is supported, the
+# following can be greatly simplified.
+gather_licenses() {
+  # shellcheck disable=SC2039  # Allow widely supported non-POSIX local keyword.
+  local module output tmp_dir acc_dir
+  output="${1}"
+  shift
+  tmp_dir="$(pwd)/gather-licenses-tmp"
+  acc_dir="$(pwd)/gather-licenses-acc"
+  mkdir "${acc_dir}"
+  cat > "${output}" <<'EOF'
+--------------------------------------------------------------------------------
+The output below is generated with `go-licenses csv` and `go-licenses save`.
+================================================================================
+EOF
+  for module ; do
+    cat >> "${output}" <<EOF
+
+go-licenses csv ${module}
+================================================================================
+EOF
+    go-licenses csv "${module}" | sort >> "${output}"
+    go-licenses save "${module}" --save_path="${tmp_dir}"
+    cp -r "${tmp_dir}"/* "${acc_dir}"/
+    rm -r "${tmp_dir}"
+  done
+  # shellcheck disable=SC2016  # Not expanding $ in single quotes intentional.
+  find "${acc_dir}" -type f | sort | xargs -L1 sh -c '
+cat <<EOF
+
+--------------------------------------------------------------------------------
+${2#${1%/}/}
+================================================================================
+EOF
+cat "${2}"
+' -- "${acc_dir}" >> "${output}"
+  rm -r "${acc_dir}"
+}
+
```

Comment:
Ideally, this `gather_licenses` function will be replaced with just `go-licenses save` calls and we let `conda-build` gather the created files then. Currently, `conda-build` only support single files under `license_file`, but once the `conda-build` issue mentioned in the comment above is resolved, I'll clean this up.